podcasts/Sharing/ShareImageView.swift

```diff
@@ -24,46 +24,74 @@ enum ShareImageStyle: CaseIterable {
             return "small"
         }
     }
+
+    var videoSize: CGSize {
```

Comment:
These are now hard coded based on the designs and these sizes are used on export. These are scaled down for preview when creating the clip.